components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -16304,6 +16304,12 @@ public void addAPIRevision(APIRevision apiRevision) throws APIManagementExceptio
                 insertOperationPolicyMappingStatement.executeBatch();
 
                 // Adding to AM_API_CLIENT_CERTIFICATE
+                String getClientCertificatesQuery = SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES;
```

Comment:
We have to correct the flow for API products revisions as well

Reply:
Added from this commit  a126b1cac2e228d29ca7bf7be8cc4fbbdda538d9